client/src/main/java/org/apache/celeborn/client/write/DataPushQueue.java

```diff
@@ -48,6 +48,7 @@ public class DataPushQueue {
   private final DataPusher dataPusher;
   private final int maxInFlightPerWorker;
   private final int shuffleId;
+  private final int appShuffleId;
```

Comment:
unused?

Reply:
no, this is a bug, it is fixed in latest commit